src/lib.rs

```diff
@@ -13,8 +13,10 @@ enum Sort {
 
 struct House {
     adress: Address,
-    capacity: i32, // For now, each geography unit contains exactly one abstract house
-    price: i32, // For now, average price inside the house
+    /// For now, each geography unit contains exactly one abstract house
```

Comment:
For documenting individual fields or structs, use the `///` above it for doc comments. https://doc.rust-lang.org/rust-by-example/meta/doc.html#doc-comments